packages/table/src/cell/cell.tsx

```diff
@@ -93,20 +93,19 @@ export type ICellRenderer = (rowIndex: number, columnIndex: number) => React.Rea
 
 export const emptyCellRenderer = () => <Cell />;
 
+/*
+ * Not a PureComponent because any children don't get included in the shallow comparison.
+ * - if we have children we always want to rerender in case the children have changed.
+ * - if we don't have children the rerender is so cheap it isn't a problem.
+ *
+ * Note that due to React's typings we can't check for the presence of children in shouldComponentUpdate() in a typesafe way.
+ */
 export class Cell extends React.Component<ICellProps, {}> {
     public static defaultProps = {
         truncated: true,
         wrapText: false,
     };
 
-    public shouldComponentUpdate(nextProps: ICellProps) {
```

Comment:
this is a concerning deletion. input from @themadcreator @cmslewis on the impact here?

Reply:
Hmm I suppose it is inconsistent with my comment above where I claim that children _can_ be memoized, so the shallow compare would be useful. Might be better to revert to the old code and add a comment making clear that the shallow compare will fail on children unless they are memoized.